### PR TITLE
Starting the bio for mrb364

### PR DESCRIPTION
**Pull request by mrb364**
## Start of a bio
- [x] get feedback from at least one team member
- [ ] add some stuff

@githubteacher thanks in advance for the lesson! #1

### CODE REVIEW

File: mrb364-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# About mrb364
+
+- Lives in Yorktown, VA
+- USAF Veteran
+- Rides a motorcycle
```
Comment (mrb364) [inline mrb364-bio.md]: Honda Interstate.

**Comment by Khillasaurus** [comment]
What kind of motorcycle?

File: mrb364-bio.md (hunk 1 above)
Comment (Khillasaurus) [inline mrb364-bio.md]: What kind of motorcycle?

**Comment by Khillasaurus** [comment]
:+1: Looks good to merge!

**Pull request merged**